packages/react/src/drafts/SelectPanel2/SelectPanel.docs.json

```diff
@@ -1,6 +1,7 @@
 {
-  "id": "selectpanel_v2",
-  "name": "SelectPanel",
+  "id": "select_panel_v2",
+  "docsId": "select_panel",
+  "name": "SelectPanel v2",
```

Comment:
@broccolini Should we call this `SelectPanel v2` or just `SelectPanel`?

Seems like [Tooltip v2 is titled just Tooltip in the docs](https://primer.style/components/tooltip/react/beta), even though [docs.json says "Tooltipv2"](https://github.com/primer/react/blob/main/packages/react/src/TooltipV2/Tooltip.docs.json#L3)

Reply:
Great catch! - And this is a [temporary hack](https://github.com/primer/design/pull/752/commits/116966476a1ff08de10aee82d758ca95014fad96) until we release https://github.com/primer/react/pull/4483 

So, we should keep the name as `SelectPanel` 😊